### PR TITLE
Use getHost instead of getContainerIpAddress

### PR DESCRIPTION
**Pull request by eddumelendez**
getContainerIpAddress is deprecated

### CODE REVIEW

**Comment by gsmet** [comment]
Hi @eddumelendez .

Thanks for this but changes to the guides need to be done there: https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc .

Once it's in the main repo, I'll merge your PR though as fixing old branches is a good idea.

**Comment by eddumelendez** [comment]
Hi @gsmet, I submitted https://github.com/quarkusio/quarkus/pull/25584 and it was merged already

**Comment by gsmet** [comment]
Sorry I missed it. I'll wait for CI and merge. Thanks!

File: _versions/2.2/guides/kafka-schema-registry-avro.adoc
Comment (gsmet) [inline _versions/2.2/guides/kafka-schema-registry-avro.adoc:616]: Just to be sure: have you checked if this method is already there in the version of Testcontainers used in 2.2?

**Comment by eddumelendez** [comment]
quarkus 2.2 uses testcontainers 1.15.3 as you can see [here](https://github.com/quarkusio/quarkus/blob/2.2.0.Final/bom/application/pom.xml#L207) and `getHost` is available is that version https://github.com/testcontainers/testcontainers-java/blob/1.15.3/core/src/main/java/org/testcontainers/containers/ContainerState.java#L58-L61

File: _versions/2.2/guides/kafka-schema-registry-avro.adoc
Comment (eddumelendez) [reply _versions/2.2/guides/kafka-schema-registry-avro.adoc:616]: quarkus 2.2 uses testcontainers 1.15.3 as you can see [here](https://github.com/quarkusio/quarkus/blob/2.2.0.Final/bom/application/pom.xml#L207) and `getHost` is available is that version https://github.com/testcontainers/testcontainers-java/blob/1.15.3/core/src/main/java/org/testcontainers/containers/ContainerState.java#L58-L61

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
Merged, thanks!

**Pull request merged**
